pkg/fission-cli/flag/flag.go

```diff
@@ -113,6 +113,7 @@ var (
 	FnConcurrency           = Flag{Type: Int, Name: flagkey.FnConcurrency, Aliases: []string{"con"}, Usage: "Maximum number of pods specialized concurrently to serve requests", DefaultValue: 500}
 	FnRequestsPerPod        = Flag{Type: Int, Name: flagkey.FnRequestsPerPod, Aliases: []string{"rpp"}, Usage: "Maximum number of concurrent requests that can be served by a specialized pod", DefaultValue: 1}
 	FnOnceOnly              = Flag{Type: Bool, Name: flagkey.FnOnceOnly, Aliases: []string{"yolo"}, Usage: "Specifies if specialized pod will serve exactly one request in its lifetime"}
+	FnSubUrl                = Flag{Type: String, Name: flagkey.FnSubUrl, Usage: "Sub URL to check if function internally support routing"}
```

Comment:
I did not understand the role of this and description is not clear?

Reply:
When user is caling `fission function test` he can only hit root endpoint of function but he wants to hit any other routes he can pass sub URL
eg.
```
$ fission function test --name nextjs-function

$ fission function test --name nextjs-function --suburl /api/hello
```

Reply:
@sanketsudake can we rename it to sub path? Sub url seems misleading as URL implies protocol+address